.github/workflows/check-full.yaml

```diff
@@ -49,6 +62,8 @@ jobs:
 
       - uses: r-lib/actions/setup-r-dependencies@v2
         with:
+          pak-version: devel
+          upgrade: "TRUE"
```

Comment:
This fixes the windows-devel build because pak fetches the latest-stringi.

Reply:
Ok but I don’t see r devel for windows failing for `main` currently. Is that still necessary?

Reply:
I tried to figure out an explanation of what `upgrade:` does and I found 
> upgrade: Whether to install the latest available versions of the dependencies. Must be an R expression. See the README for details if you need quoting. Defaults to FALSE.

So does that mean that we might install source packages but that's better than outdated binaries because of incompatibilities between the latest binary of package A and an outdated binary of package B?

Reply:
Yes, exactly, provided they are available for that particular R version.